app/src/main/java/com/example/myapplication/SecondFragment.java

```diff
@@ -78,6 +78,7 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
             animation.start();
             UIHelpers.darkMode = !UIHelpers.darkMode;
             UIHelpers.lightDark(v, UIHelpers.darkMode);
+            UIHelpers.playHowlSound(this.getContext());
```

Comment:
Those four lines are the same in many fragments. Consider moving them to a method in UIHelpers:

    public static void darkModeToggle(ObjectAnimator animation, Context context) {
        animation.start();
        darkMode = !darkMode;
        lightDark(darkMode);
        playHowlSound(context);
    }

Reply:
Ok ;)

Reply:
Done